### PR TITLE
Adding renoweb_dk

### PR DESCRIPTION
**Pull request by r-poulsen**
First time contributor adding support for RenoWeb, used by several Danish municipalities.

Added the service .py and .md, updated README.md and info.md.

Test looks good:

```
$ ./test_sources.py -s renoweb_dk --weekday 
Testing source renoweb_dk ...
  found 26 entries for test_01
  found 207 entries for test_02
  found 350 entries for test_03
```

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+_LOGGER = logging.getLogger('waste_collection_schedule.renoweb_dk')
+
+
+class Source:
+    """ Source class for RenoWeb """
+
+    _api_url: str
+    _address_id: int
+
+    def __init__(self, **kwargs):
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py:41]: Please do not use **kwargs but each argument separately (use = None for optionals) The user should instantly get an error when passing wrong arguments

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            response.raise_for_status()
+
+            _LOGGER.debug(
+                "Address '%s'; id %s",
+                json.loads(response.json()['d'])['list'][0]['label'],
+                json.loads(response.json()['d'])['list'][0]['value']
+            )
+
+            self._address_id = json.loads(response.json()['d'])[
+                'list'][0]['value']
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py:89]: You could put this in a separate method to declutter the fetch method

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    _address_id: int
+
+    def __init__(self, **kwargs):
+        _LOGGER.debug("Source.__init__(); %s", kwargs)
+        if kwargs and "municipality" in kwargs:
+            self._api_url = (
+                'https://'
+                + kwargs["municipality"].lower()
+                + '.renoweb.dk/Legacy/JService.asmx/'
+            )
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py:48]: I'd prefer:
```python
# Global constant
API_URL = "https://{municipality}.renoweb.dk/Legacy/JService.asmx/{{endpoint}}"
# init:
self._api_url = API_URL.format(municipality=kwargs["municipality"].lower())

# Fetch: 

url=self._api_url.format(endpoint="Adresse_SearchByString")
url=self._api_url.format(endpoint="GetAffaldsplanMateriel_mitAffald")
url=self._api_url.format(endpoint="GetCalender_mitAffald")

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+URL = "https://renoweb.dk"
+
+TEST_CASES = {
+    "test_01": {
+        "municipality": "frederiksberg",
+        "address": "Roskildevej 40",
+    },
+    "test_02": {
+        "municipality": "htk",
+        "address": "🤷",
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py:23]: address is not needed here

**Review by 5ila5 — CHANGES_REQUESTED** [review]

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py (hunk 1 above)
Comment (r-poulsen) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py:41]: Done, now also raises an exception if neither address or address_id is provided

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py (hunk 2 above)
Comment (r-poulsen) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py:89]: Refactored into a property

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py (hunk 3 above)
Comment (r-poulsen) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py:48]: Done

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py (hunk 4 above)
Comment (r-poulsen) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/renoweb_dk.py:23]: Done

**Comment by 5ila5** [comment]
Thanks for your contribution.

Did some minor reformatting (based on .pre-commit-config.yaml)

**Review by 5ila5 — APPROVED** [review]

**Pull request merged**